### PR TITLE
Feat: Encrypted Query Example

### PR DESCRIPTION
**Pull request by riccardopersiani**
Please @gskapka, could you review the code? 

The datasource is not encrypted because, for now, the datasource encryption is not available.

### CODE REVIEW

**Comment by D-Nice** [comment]
i just saw now that PR https://github.com/provable-things/ethereum-examples/pull/60 implements a number of these suggestions across the different examples. 

I would still ask for some of the reprphrasing suggestions to be included. And then I'll review once more. We can leave the oraclize functions in, but the provable functions have to be renamed, and I would have them each CI'd.

File: solidity/truffle-examples/encrypted-query/contracts/EncryptedQuery.sol [hunk 1]
```diff
@@ -0,0 +1,47 @@
+pragma solidity >= 0.5.0 < 0.6.0;
+
+import "./oraclizeAPI.sol";
```
[the remark this answers is not in the document]
Comment (riccardopersiani) [reply solidity/truffle-examples/encrypted-query/contracts/EncryptedQuery.sol]: Done.

File: solidity/truffle-examples/encrypted-query/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+&nbsp;
+
+This repo is to demonstrate how you would set up an Provable smart-contract development environment using Truffle & the Ethereum-Bridge to do most of the heavy lifting for you. Head on over to the `./test` folder to examine the javascript files that thoroughly test the smart-contract, which latter you will find in `./contracts`.
+
+## :squirrel: _Query Encryption_
+
+**1)** Decide the Provable query you want to encrypt, the one in this example is:
+
+```
+oraclize_query(
```
[the remark this answers is not in the document]
Comment (riccardopersiani) [reply solidity/truffle-examples/encrypted-query/README.md]: Done.

**Comment by D-Nice** [comment]
Looking good to me atm, I will add CI on top just to ensure it is all running as expected before final approval and merge.

**Pull request merged**